docs/conf/portland/2025/cfp.rst

```diff
@@ -60,12 +55,13 @@ That said, there are some more helpful and concrete pointers below.
 
 **Diverse group of speakers**
 
-We welcome talks from first-time speakers, from industry experts, and from everyone in between.
-Whatever your background and experience, we prefer hearing about new approaches rather than about tried-and-tested technology.
-We especially welcome talks from underrepresented groups within the tech community.
+We welcome first-time speakers, industry experts, and everyone in between. We especially welcome talks from underrepresented groups within the tech community.
+
 We want to hear a variety of viewpoints, so we limit speakers to two talks in any four year period at each location.
 
-If you're a WTD conference organizer, please only submit talks to conferences you're not actively organizing.
+**Single vs. co-speakers**
```

Comment:
```suggestion
**Number of speakers**
```

Reply:
Though I'd rather leave in the strong additions I put in on L68:70 - edits welcome